.asf.yaml

```diff
@@ -0,0 +1,37 @@
+  
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+github:
+  description: An observability database aims to ingest, analyze and store Metrics, Tracing and Logging data.
+  homepage: https://skywalking.apache.org/
+  labels:
+    - skywalking
+    - observability
+    - apm
+    - database
+  enabled_merge_buttons:
+    squash:  true
+    merge:   false
+    rebase:  false
+  protected_branches:
+    main:
+      required_status_checks:
+        contexts:
+          - Continuous Integration
```

Comment:
These still not allows direct push to main branch

Reply:
Yes, but they can merge the PR through pull request alternatively.

Reply:
If we don't want to allow direct push to main branch but allow merging committers' own PRs, this is OK to me

Reply:
Currently require CI passed and a committer merge(squash only).